### ISSUE TITLE
Problem Starting Picket on Ubuntu

### ISSUE THREAD

**Issue by erp12** [Z5T1/Picket#1]
When trying to start a picket server on my Ubuntu machine with Minecraft 1.8.1, I get the following messages:

<pre><code>Starting the Minecraft server and Picket
There are several suitable screens on:
    7391.picket (01/22/2015 10:40:43 PM)    (Detached)
    6924.picket (01/22/2015 10:34:57 PM)    (Detached)
    3644.picket (01/21/2015 12:12:06 AM)    (Detached)
Use -S to specify a session.
./picket: error while loading shared libraries: libpcap.so.1: cannot open shared object file: No such file or directory</code></pre>

Here is the contents of my Picket.config:

<pre><code>######################### Picket Configuration #########################

# A comma separated list of plugins to load
plugins="foo"

# The maximum number of players allowed to connect
max_players=20

# The port the minecraft server is set to run on
port=25565

# The screen to run the minecraft server in. If you are running multiple
# instances of picket, make sure each instance is set to run in a
# different screen.
screen=picket

# The interface to listen for picket on. In most cases, this should be
# set to your primary network interface.
interface=lo

# The command to use to start the minecraft server
server_command="java -jar -Xmx512M minecraft_server.1.8.1.jar nogui"</code></pre>

Chances are I have missed something, like a dependancy. Let me know if you need any more information, and I hope this helps :)

(This is edster3194 from reddit)

**Comment by Z5T1** [comment]
Yes you're missing a dependency. Try installing the package tcpdump.

**Comment by erp12** [comment]
So after finally getting around to trying to fix this problem, I discovered that I do actually have tcpdump, but by default it tries to use link type BLUETOOTH_HCI_H4_WITH_PHDR, which doesn't support packet printing.

To fix this, I disabled bluetooth on my machine. Then when I ran tcpdump on its own, I got the following error:
<code>tcpdump: no suitable device found</code>
Also, I get the same error when starting picket from my origional post.

Any advice? I don't have any expirience working with packets, so I don't really know what I am looking for.

**Comment by Z5T1** [comment]
What network devices do you have on your computer? Both tcpdump and Picket require you to specify a network device. Check the output of `ifconfig`. Did you specify one of those devices listed there as your interface in `picket.config`? If you're just testing it on your local machine, you should be able to use the loopback interface (usually it's something along the lines of `lo`).
